flink-runtime/src/main/java/org/apache/flink/runtime/state/filesystem/FsCheckpointStreamFactory.java

```diff
@@ -247,15 +247,13 @@ public long getPos() throws IOException {
 			return pos + (outStream == null ? 0 : outStream.getPos());
 		}
 
-		@Override
-		public void flush() throws IOException {
+		private void flushToFile() throws IOException {
```

Comment:
We can make this public, let the tests call it directly.

Reply:
Done.